src/Rooms.js

```diff
@@ -1,8 +1,19 @@
 class Rooms {
```

Comment:
Same comment regarding singular class title, `class Room {`

Reply:
Thank you. I will change the name